operator/provider_azure.go

```diff
@@ -0,0 +1,25 @@
+//+build operator_azure
```

Comment:
The build tags can live on line 16

Reply:
Ah, yes, I see we have build tags below license headers in other packages.